### ISSUE TITLE
Objects leaving trails in Firefox with antialias and preserveDrawingBuffer

### ISSUE THREAD

**Issue by marekjedrzejewski** [mrdoob/three.js#17803]
##### Description of the problem

When background is one-color, objects leave trails like this
![Effect](https://i.imgur.com/I1VhWgO.png)
with
```
{ antialias: true, preserveDrawingBuffer: true }
```
in WebGLRenderer. I only observed this in Firefox on Arch Linux with RX 570. Tried with other browsers, MacBook, Windows in VM, laptop with Fedora and integrated intel graphics, and everything else works just fine. I'll also try with live ubuntu on this machine to check and update this issue afterwards.
EDIT: Ubuntu 19.10 live on the same machine also has this issue. 

https://jsfiddle.net/w0vejxat/

##### Browser

- [ ] All of them
- [ ] Chrome
- [x] Firefox
- [ ] Internet Explorer

##### OS

- [ ] All of them
- [ ] Windows
- [ ] macOS
- [x] Linux
- [ ] Android
- [ ] iOS

##### Hardware Requirements (graphics card, VR Device, ...)
Maybe Radeon RX570?

**Comment by mrdoob** [comment]
That looks like a browser bug.
Do you mind reporting it to Mozilla instead?

**Comment by Mugen87** [comment]
> Do you mind reporting it to Mozilla instead?

Related link: https://bugzilla.mozilla.org/home

**Comment by marekjedrzejewski** [comment]
I investigated it further: when I switched Firefox version, nothing changed. Didn't try to change mesa, because this can get messy because of all the dependencies, but with Ubuntu 18.4.3 live, which has Mesa 19.0, the problem also doesn't show up. Ubuntu 19.10 and Arch with Mesa 19.2 both have it. But it only shows up in Firefox.

I'll report to mesa gitlab, because while it only shows up in Firefox, it was caused by some change there.
